### PR TITLE
Add Hatena Blog Loader

### PR DESCRIPTION
**Pull request by morugu**
[Hatena Blog](https://hatenablog.com/) is a famous blogging platform in Japan.  
You might be able to tailor the LLM to create a conversation experience as if you're talking to yourself.

### CODE REVIEW

**Comment by jerryjliu** [comment]
amazing! will take a look soon :)

File: loader_hub/hatena_blog/base.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+class Article():
+    def __init__(self) -> None:
+        self.title = ''
+        self.content = ''
+        self.published = ''
+
+
+class HatenaBlogReader():
+    """Hatena Blog reader
```
Comment (jerryjliu) [inline loader_hub/hatena_blog/base.py:17]: ubernit: add period at end of docstring first sentence

File: loader_hub/hatena_blog/base.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
+"""Hatena Blog reader."""
+
+from typing import List
+from llama_index.readers.schema.base import Document
+
+ATOM_PUB_ENTRY_URL = '{root_endpoint}/entry'
+
+
+class Article():
```
Comment (jerryjliu) [inline loader_hub/hatena_blog/base.py:9]: nit: no need for the parentheses

File: loader_hub/hatena_blog/base.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        while True:
+            res = self.get_articles(page_url)
+            articles += res.get('articles')
+            page_url = res.get('next_page')
+            if page_url is None:
+                break
+
+        return articles
+
+    def get_articles(self, url: str):
```
Comment (jerryjliu) [inline loader_hub/hatena_blog/base.py:54]: add return type annotation (-> Dict)

**Review by jerryjliu — CHANGES_REQUESTED** [review]
looks pretty good! main comment is could you add this as an entry to `library.json`?

**Comment by morugu** [comment]
@jerryjliu Appreciate the review! Updated code as suggested.

**Review by jerryjliu — APPROVED** [review]

**Pull request merged**